<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatusControlOptionsDlg</class>
 <widget class="QDialog" name="StatusControlOptionsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>381</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="DebugLevelLabel">
        <property name="text">
         <string>Debug Level</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="DebugLevelEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="HistoryLabel">
        <property name="text">
         <string>History Length</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="HistoryEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="ShowMsgOrigin">
      <property name="text">
       <string>Indicate Message Origin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="SystemSpecificsTable"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="okBtn">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelBtn">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
